server/db/index.js

```diff
@@ -0,0 +1,47 @@
+const mysql = require('mysql');
```

Comment:
Not sure what best practice is but I have a separate directory for my database instead of nesting my db inside of server directory.